helpdesk_mgmt_timesheet/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "Helpdesk Timesheet",
+    "summary": """
+        Add Timesheets to the Helpdesk""",
+    "category": "After-Sales",
+    "version": "11.0.1.0.0",
+    "depends": ["helpdesk_mgmt",
+                "hr_timesheet"
```

Comment:
With the purpose of diminishing the number of line changes in future commits every line should contain a `,` at the end